internal/trigger/http/rest.go

```diff
@@ -113,7 +115,14 @@ func (trigger *Trigger) requestHandler(writer http.ResponseWriter, r *http.Reque
 	appContext := trigger.serviceBinding.BuildContext(envelope)
 
 	defaultPipeline := trigger.serviceBinding.GetDefaultPipeline()
-	messageError := trigger.serviceBinding.ProcessMessage(appContext.(*appfunction.Context), envelope, defaultPipeline)
+	targetData, decodeErr, _ := trigger.serviceBinding.DecodeMessage(appContext.(*appfunction.Context), envelope)
+	if decodeErr != nil {
+		writer.WriteHeader(decodeErr.ErrorCode)
+		_, _ = writer.Write([]byte(decodeErr.Err.Error()))
```

Comment:
is this fine to ignore the err returned here?

Reply:
No, it should be logged.

Reply:
fixed